script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -466,12 +513,14 @@ That's right.
 [sha:311b1aa8352964290b27216bcd804f98dc2509cf]{
 -- Page 450, Offset 28329.
 -- たとえばあの地下ガレージの吸血鬼たち。連中はⅢ階梯より上だった。ロアが死んでいたとしても、そこに矛盾はない。
-The vampires in that underground garage for instance. Those guys were above rank III. There's nothing contradictory, even if Roa's dead.
+Take the vampires in that underground garage, for instance.
+ They were definitely all above Rank III.
+ Everything can still be explained, even if Roa's dead.
 }
 [sha:0f3982fa9e2bdd5715ee9a3ba506e395e9eb4bb5]{
 -- Page 451, Offset 28330.
 -- 「……ニュースは、ありましたか？」
-"Was there... any report?"
+"The news... Have there been any reports?"
```

Comment:
Hmm, that feels _more_ disconnected to me. It's a strange place to break that sentence. "The news... was there anything on the news?!" seems more natural imo.

Reply:
Then just remove the ellipsis.

Reply:
I mean I'd prefer to keep the dramatic pause, since there kinda is one in the VA. Hence adjusting the sentence so that the pause is in a place that flows well in EN.

Reply:
Is it that pronounced? Well, go with 

>The news... was there anything on the news?

that you suggested then.